### ISSUE TITLE
FHIR Exception

### ISSUE THREAD

**Issue by jawalonoski** [synthetichealth/synthea#445]
```
java.lang.NumberFormatException
        at java.math.BigDecimal.<init>(BigDecimal.java:494)
        at java.math.BigDecimal.<init>(BigDecimal.java:383)
        at java.math.BigDecimal.<init>(BigDecimal.java:806)
        at java.math.BigDecimal.valueOf(BigDecimal.java:1274)
        at org.hl7.fhir.dstu3.model.DecimalType.setValue(DecimalType.java:165)
        at org.hl7.fhir.dstu3.model.Quantity.setValue(Quantity.java:335)
        at org.mitre.synthea.export.FhirStu3.mapValueToFHIRType(FhirStu3.java:1621)
        at org.mitre.synthea.export.FhirStu3.observation(FhirStu3.java:1571)
        at org.mitre.synthea.export.FhirStu3.convertToFHIR(FhirStu3.java:237)
        at org.mitre.synthea.export.Exporter.export(Exporter.java:40)
        at org.mitre.synthea.engine.Generator.generatePerson(Generator.java:349)
        at org.mitre.synthea.engine.Generator.lambda$run$0(Generator.java:197)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by jawalonoski** [comment]
The only way that `BigDecimal.valueOf` throws a `NumberFormatException` is if the the `double` arguments is Infinity or NaN. Double values in java default to `0.0d` and there are no obvious references in the code to setting any double to NaN or Infinity. Unable to recreate so far....

**Comment by tiloc** [comment]
Maybe related to what I mentioned in Issue #291? People don't get all their vital signs populated at birth and trying to create observations from unpopulated vital signs could result in trouble?

**Comment by jawalonoski** [comment]
Presumed fixed by #448. Not encountered since.
